NEWS

```diff
@@ -13,6 +13,7 @@
 * Fix build issues in gtest-all.cc and sock.c with GCC11
 * Fix performance degradation with cuda memory on self endpoint
 * Fix bug in JUCX listener connection handler.
+* Fix bug in short reply protocol.
```

Comment:
Can you please change to `Fixes in short reply protocol`. 

I know you have been following the format of news posted above, but those are no good and I will be rewriting those to follow our usual news format. thanks

Reply:
Fixed.